net-irc/inspircd/files/inspircd-r3.initd

```diff
@@ -0,0 +1,47 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+INSPIRCD_USER=${INSPIRCD_USER:-"inspircd"}
```

Comment:
technically, it is more "correct" in shell to set a colon `:` at the beginning of a line that defines a default variable. For instance:
```
: ${foo-:"bar"}
```

Reply:
Like this?